client/src/Components/Comment/Comment.js

```diff
@@ -51,14 +57,27 @@ const Comment = (props) => {
 						} else {
 							status = 0;
 						}
-						return <Comment reactions={props.reactions} children={comment.children} depth={props.depth + 1} original={props.original} key={comment._id} id={comment._id} body={comment.body} upvotes={comment.upvotes} date={comment.date} status={status} author={comment.author}/>;
+						return (
+							<Comment
+								reactions={props.reactions}
+								children={comment.children}
+								depth={props.depth + 1}
+								original={props.original}
+								key={comment._id} id={comment._id}
+								body={comment.body}
+								upvotes={comment.upvotes}
+								date={comment.date}
+								status={status}
+								author={comment.author}
+							/>
+						);
 					})
 				)
 				}
 				{(showChildren === 'shown' && comments) &&
 				(
 					// react converts 1 elemenet object arrays to just that object in state
-					comments.map((comment, idx) => {
+					comments.sort((a, b) => b.upvotes - a.upvotes).map((comment, idx) => {
```

Comment:
Think about sorting on DB side.
The common priority - move these type of operations further:
- db
- server
- at least use memoization